mods/ra/weapons/missiles.yaml

```diff
@@ -461,6 +473,7 @@ SCUD:
 		Inaccuracy: 213
 		Image: V2
 		LaunchAngle: 62
+		Shadow: True
```

Comment:
You need to delete the `Shadow: false` 6 lines above this.  This is what causes fort lonestar to fail.